azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -530,6 +546,18 @@ def execute(self, database: str, query: str, properties: ClientRequestProperties
             return self.execute_mgmt(database, query, properties)
         return self.execute_query(database, query, properties)
 
+    async def execute_async(self, database: str, query: str, properties: ClientRequestProperties = None) -> KustoResponseDataSet:

```

Comment:
how does python handle the async keyword if no asynio is installed?

Reply:
Asyncio comes integrated with python 3.5+